client/lib/signup/step-actions.js

```diff
@@ -333,7 +333,7 @@ export function createAccount(
 				if ( errors ) {
 					callback( errors );
 				} else {
-					callback( undefined, response );
+					callback( undefined, pick( response, [ 'username', 'bearer_token' ] ) );
```

Comment:
This is fine for this A/B test, but let's not keep that once it is over.

Reply:
Why?

Reply:
No, we should keep that part, this object has to match the `providesDependencies` config for this step as defined in `client/signup/config/steps.js`. The problem doesn't come from this PR it comes from our recent change in the format of the `/users/social/new` response. This change should ensure it won't happen again.